commands/goToDist.py

```diff
@@ -30,9 +30,13 @@ def initialize(self) -> None:
     def execute(self) -> None:
         """Called every time the scheduler runs while the command is scheduled."""
         self.dist = abs(self.drive.getRightEncoder()) - self.startingDistance
-        self.totalOffset = self.targetDist - self.dist
-        self.speed = self.pid.calculate(self.targetDist / 10_000, self.dist / 10_000)
-        self.drive.drive(-1 * self.speed, -1 * self.speed)
+        if(self.targetDist >= 0):
```

Comment:
Comment on what we're doing here

Reply:
If you switced the math order you can avoid this if statment. Since the sign of "targetDist" would handle it.

Reply:
```suggestion
        #TODO investigate https://github.com/Raptacon/Robot-2023/pull/152/files#r1148391038 
        if(self.targetDist >= 0):
```